### PR TITLE
Am feature navigation

### PR DESCRIPTION
**Pull request by amarquesmoura**
this adds the icons to App header and Drawer header

### CODE REVIEW

**Review by teelau — DISMISSED** [review]

**Comment by teelau** [comment]
Thanks for changing the width to be dynamic!

File: client/js/navigation/config.js [hunk 1]
```diff
@@ -23,7 +22,26 @@ export const sharedNavigationOptions = navigation => ({
   },
   headerLeft: (
     <TouchableOpacity onPress={() => navigation.toggleDrawer()}>
-      <Icon name="md-menu" size={25} color="white" style={{ margin: 15 }} />
+      <Image
+        source={require("../assets/images/hamburgermenu.png")}
```
Comment (siddharthparmar7) [inline client/js/navigation/config.js]: Use react-native-vector-icons instead of an image.

File: client/js/navigation/config.js [hunk 2]
```diff
@@ -23,7 +22,26 @@ export const sharedNavigationOptions = navigation => ({
...
+      <Image
+        source={require("../assets/images/bikelockerlogo.png")}
+        style={{ width: 40, height: 40, marginBottom: 5 }}
+      />
+    </TouchableOpacity>
+  ),
+  headerRight: (
+    <TouchableOpacity>
+      <Image
+        source={require("../assets/images/location.png")}
```
Comment (siddharthparmar7) [inline client/js/navigation/config.js]: Use react-native-vector-icons instead of an image.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make the changes. Thank you.

**Comment by amarquesmoura** [comment]
Done.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Nice work with navigation. Your Profile screen code should not be in this branch. Please make the changes.

**Comment by amarquesmoura** [comment]
Done.

File: client/js/navigation/config.js (hunk 1 above)
Comment (amarquesmoura) [reply client/js/navigation/config.js]: Done.

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**